x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -22657,7 +22657,6 @@
     "xpack.securitySolution.alertDetails.summary.readLess": "表示を減らす",
     "xpack.securitySolution.alertDetails.summary.readMore": "続きを読む",
     "xpack.securitySolution.alertDetails.threatIntel": "Threat Intel",
-    "xpack.securitySolution.alerts.badge.readOnly.text": "読み取り専用",
```

Comment:
Do we need to do anything about these translations? It looks like only the path has changed from `xpack.securitySolution.alerts.badge.readOnly.text` to `xpack.securitySolution.badge.readOnly.text`, but I don't know if that means the translation might still need to be updated.

Reply:
I beliiieve that if the path changes it is treated as a new translation needed. I'll double check there.